scripts/jQuery.leanModal2.js

```diff
@@ -0,0 +1,91 @@
+// jQuery.leanModal2.js
+// MIT Licensed by eustasy http://eustasy.org
+// Based on leanModal v1.1 by Ray Stone - http://finelysliced.com.au
+
+// Wrap in an anonymous function.
+(function($){
+
+    // Function: Fade out the Overlay and a passed identifier
+    function leanModal_Close(modal_id) {
+        $('.js-target-jquery-leanmodal-overlay').fadeOut(300);
+        $(modal_id).fadeOut(200);
+
+        $('body').css({ 'overflow': 'visible' });
+    }
+
+    // Define a new Extension.
+    $.fn.extend({
+        leanModal: function(options) {
+
+            // Set some Defaults.
+            var defaults = {
+                overlayOpacity: 0.6,
+                closeButton: false
+            };
+
+            // Merge in the passed options.
+            options = $.extend(defaults, options);
+
+            // If there isn't an overlay, add one.
+            if ( $('.js-target-jquery-leanmodal-overlay').length == 0 ) {
+                var overlay = $('<div class="js-target-jquery-leanmodal-overlay"></div>');
+                $('body').append(overlay);
+            }
+
+            // FORLINK For each targetted link.
+            return this.each(function() {
+
+                $(this).css({ 'cursor': 'pointer' });
+
+                $(this).unbind('click').click(function(e) {
+
+                    // IFHREF Fetch the Modal_ID
+                    if ( $(this).attr('href') ) {
+                        var modal_id = $(this).attr('href');
+                    // IFHREF Fall back to if no href data-open-modal.
+                    } else if ( $(this).attr('data-modal-id') ) {
+                        var modal_id = $(this).attr('data-modal-id');
+                    } else {
+                        return false;
+                    } // IFHREF
+
+                    // Set the function to close the overlay if you click it.
```

Comment:
I'm not sure if we want clicking on the overlay to close the modal. If not, we want this gone.